skietbaan_fe/src/components/LoginStyles.css

```diff
@@ -38,8 +38,28 @@ body{
   top:25%
 }
 }
+input.invalid{
+  border:1px solid red;
+}
+.invalid-message{
+  color:red;
+  font-size: 12px;
+}
+button{
+  border-radius: 30px !important;
```

Comment:
Avoid using `!important` in css. As is, this will affect all the buttons in your app, so what happens when you want to style another button differently??? Rather use a more specific [css-selector](https://www.w3schools.com/cssref/css_selectors.asp) to make sure your only style what you intend to style.

Reply:
Alright removed it